scripts/ci-tool.py

```diff
@@ -310,20 +313,26 @@ def main():
 
     # define all the build roots
     project_root = abspath(dirname(dirname(__file__)))
-    build_root = join(project_root, '{}{}'.format(
-        args.build_path_prefix, args.build_type.lower()))
+    build_root = join(
+        project_root, f'{args.build_path_prefix}{args.build_type.lower()}')
     if args.force_build_folder:
         build_root = abspath(args.force_build_folder)
 
     options = {
-        'CMAKE_BUILD_TYPE': args.build_type
+        'CMAKE_BUILD_TYPE': args.build_type,
     }
 
     if args.metrics:
         options['FETCH_ENABLE_METRICS'] = 1
 
+    if args.lint:
+        options['CMAKE_EXPORT_COMPILE_COMMANDS'] = 1
```

Comment:
This option should always be enabled in the crake configuration so we shouldn’t need to set it here

Reply:
Done.